keps/sig-storage/1979-object-storage-support/README.md

```diff
@@ -149,22 +149,17 @@ metadata:
   finalizers:
     - cosi.io/finalizer [2]
 spec:
-  protocol:
-    name: [3]
-    version: [4]
-  bucketPrefix: [5]
-  bucketClassName: [6]
-  bucketInstanceName: [7]
+  bucketPrefix: [3]
+  bucketClassName: [4]
+  bucketInstanceName: [5]
```

Comment:
the type is `Bucket` not `BucketInstance`, so it should be bucketName ?

Reply:
> so it should be bucketName ?

Technically yes. Instance was added to the name to be more precise about this field referring to the `Bucket` instance name and not the name of the underlying bucket.

Reply:
The established convention is `<Kind>Name` - I'm loathe to buck the trend

Reply:
Field name changed to `bucketName`.